translator/ml_translatorLib.sml

```diff
@@ -424,9 +424,9 @@ val basic_theories =
    ["alist", "arithmetic", "bag", "bitstring", "bit", "bool",
     "combin", "container", "divides", "fcp", "finite_map", "float",
     "fmaptree", "frac", "gcdset", "gcd", "ind_type", "integer_word",
-    "integer", "integral", "list", "llist", "marker", "measure",
-    "numeral_bit", "numeral", "numpair", "numposrep", "num", "one",
-    "operator", "option", "pair", "path", "patricia_casts",
+    "integer", "integral", "list", "llist", "marker", "machine_iee",
```

Comment:
This looks wrong... doesn't "machine_iee" need another "e"? Why wouldn't the regression test catch this?

Reply:
Interesting. Here is my "suggested fix":
```suggestion
    "integer", "integral", "list", "llist", "marker", "machine_ieee",
```

Could it be that it is not caught by the regression test because the variable is only used in function `full_name_of_type` below, which searches for the `thy_name` if it is not in the list?